sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -80,6 +80,9 @@ static void *video_thread_function(void *data)
 {
   struct video_config_t *vid = (struct video_config_t *)data;
 
+  char print_tag[80];
+  sprintf(print_tag, "video_thread-%s", vid->dev_name);
```

Comment:
Besides the niceness level, it would be great if the priorities can be set between different vision functions. For example, textons should run at 5Hz and optical flow at 30Hz. With a periodic function this was / (is?) possible, but the vision process is a bit differently arranged now, with all vision functions added with cv_add.